api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirAllergyIntoleranceDaoImpl.java

```diff
@@ -45,7 +46,7 @@
 	@Inject
 	private FhirGlobalPropertyService globalPropertyService;
 	
-	private List<String> severityConceptUuids;
+	private Map<String, String> severityConceptUuids;
```

Comment:
I don't think we want this as a class variable. It should just be a local variable in the one function taht uses it.